doc/mgr/diskprediction.rst

```diff
@@ -0,0 +1,113 @@
+=====================
+DiskPrediction plugin
+=====================
+
+Disk Prediction plugin is used to collect disk information from Ceph OSD and
+send DiskPrediction server. The disk information includes the following.
+
+- Ceph status
+- I/O operations
+- I/O bandwidth
+- OSD status
+- OSD physical disk smart data
+- Storage utilization
+
+Enabling
+========
+
+Run the following command to enable *diskprediction* module in the Ceph
+environment:
+
+::
+
+    ceph mgr module enable diskprediction
+
+Connection settings
+-------------------
+
+The connection settings can be configured on any machine with the proper cephx
```

Comment:
I would skip the "any machine with proper cephx credentials" sentence -- this is just the normal Ceph CLI, so it's fair to assume the user already has a machine where they can use it.